site/data/docs/migrating-to-02.mdx

```diff
@@ -6,6 +6,26 @@ title: Migrating to >=0.2
 
 ## Migrating RainbowKit
 
+### 0.4.x Breaking changes
+
+RainbowKit has updated the `wagmi` peer dependency to `0.5.x`.
+
+Follow the steps below to migrate.
+
+#### 1. Upgrade dependencies
+
+Upgrade RainbowKit and wagmi to their latest version:
+
+```bash
+npm i @rainbow-me/rainbowkit@^0.4.0 wagmi@^0.5.0
+```
+
+#### 2. Check for breaking changes in `wagmi`
+
+If you use `wagmi` hooks in your application, you will need to check if your application has been affected by the breaking changes in `wagmi`.
+
+[You can see their migration guide here](https://wagmi.sh/docs/migrating-to-03#05x-breaking-changes).
+
```

Comment:
Technically, it is still correct since consumers could be upgrading from 0.2 to 0.3/0.4. Open to other suggestions of titles though (like: "Migrating to newer versions" or smth)!

Reply:
Could just be "Migration guides"?

Reply:
i likey